source/meetings/2022/december/index.html.md

```diff
@@ -0,0 +1,93 @@
+---
+updated_by:
+  email: paolo.fabbri@lrug.org
+  name: Paolo Fabbri
+created_by:
+  email: paolo.fabbri@lrug.org
+  name: Paolo Fabbri
+category: meeting
+title: December 2022 Meeting
+published_at: 2022-11-13 21:00:00 +0000
+created_at: 2022-11-13 21:00:00 +0000
+status: Published
+hosted_by:
+  - :name: Zinc
+
+---
+
+The December 2022 meeting of LRUG will be on *Monday the 12th of
+December*, from _6:00pm_ to _8:00pm_ (meeting starts at _6:30pm_).
+
+**👪 in person meeting alert 👪**
+
+and with the December meetup it will be our fourth in person meetup of the 2022; all thanks to the helps of the amazing people at [Zinc](https://www.zinc.vc) at [their offices][zinc-venue], on Eversholt Street. [Full venue and
+registration details are given below](#dec22registration).
+
+## Agenda
+
+### Mutation testing - study case
+
+[Adam Piotrowski](https://twitter.com/pan_sarin) (also [here](https://www.linkedin.com/in/adamsarin)) says:
+
+> Let's talk about why and how we measure our test coverage. If you are
+> using line test coverage measurement and you are happy with it, please let
+> me show you some differences and examples of line TC vs mutation TC.
+
+### Heaping on the Complexity
+
+[Matt Valentine-House](https://ruby.social/@eightbitraptor) says:
+
+> Join me on a journey through Ruby's Garbage Collector!
+> 
+> In this talk I'll teach you some of the details about how the Ruby
+> interpreter manages memory. I'll introduce a project my team and I are
+> working on that aims to make Ruby faster by improving its memory
+> efficiency, and then we'll talk about how our implementation broke
+> Garbage Collection.
+
+> After that we'll go on a journey together, through some weeds, and
+> taking a few bad turns until we finally emerge with a few PR's that
+> not only Fix GC, but make our project better too.
+
+###  What does "high priority" mean? The secret to happy queues
+
+[Daniel Magliola]()
```

Comment:
Maybe use the same Twitter link as went he last spoke? https://mobile.twitter.com/dmagliola

Reply:
I looked around and didn't found it and forgot about that